pkgs/development/python-modules/flask-wtf/default.nix

```diff
@@ -19,4 +19,8 @@ buildPythonPackage rec {
     maintainers = [ maintainers.mic92 ];
     homepage = "https://github.com/lepture/flask-wtf/";
   };
+
+  passthru.wtforms2 = flask_wtf.override {
+    wtforms = wtforms.wtforms2;
+  };
```

Comment:
```suggestion
```
I don't think we even need this.

Reply:
Well unless you're applying an overlay to the python package set, anything that indirectly depends on wtforms must also be overridden to keep it out of the closure.

But yes, this can instead be overridden manually everywhere `flask_wtf` is used, if that's what you mean.

Reply:
You want to use something like that https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/misc/privacyidea/default.nix#L6-L39 in the final package.